native/machine-coding/steps-bar/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="../../helpers/navbar.js" type="module"></script>
```

Comment:
header to be imported. Also, meta tags are no more needed

Reply:
hi i fixed the issue but i don't get you change number of steps meaning ? actually it basically progress steps bar. each next click will progress the bar and goes like this.